src/script/lua_api/l_object.cpp

```diff
@@ -1724,6 +1724,19 @@ int ObjectRef::l_set_sky(lua_State *L)
 			read_color(L, -1, &sky_params.bgcolor);
 		lua_pop(L, 1);
 
+		lua_getfield(L, 2, "body_orbit_tilt");
+		if (!lua_isnil(L, -1)) {
+			sky_params.body_orbit_tilt = static_cast<float>(luaL_checknumber(L, -1));
+			if (sky_params.body_orbit_tilt < -60.0f) {
+				sky_params.body_orbit_tilt = -60.0f;
+				warningstream << "body_orbit_tilt value below -60.0 - corrected.\n" << std::endl;
+			} else if (sky_params.body_orbit_tilt > 60.0f) {
+				sky_params.body_orbit_tilt = 60.0f;
+				warningstream << "body_orbit_tilt value above 60.0 - corrected.\n" << std::endl;
+			}
```

Comment:
we don't usually warn for these things
```suggestion
			sky_params.body_orbit_tilt = core::clamp(readParam<float>(L, -1), -60.0f, 60.0f);
```

Reply:
So, omit the warning and just clip the value to it's max?

Reply:
Yes, just rangelim -60, 60 should do the job. It's better to document it's clipped then to issue a warning on every API call.